target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4018-whw01-v1.dts

```diff
@@ -0,0 +1,237 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
```

Comment:
dts-v1 can be dropped, it's already in qcom-ipq4018.dtsi. (This does not require any testing.)

Reply:
Removed.